cgkineo/adapt-xAPI
# Breaking: ES6 Upgrade (fixes: #13)

[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)

### Update
* Removing favorite, unfavorite & modifying preferred statements
* ES6 Upgrade of all JS

### New
* New verb _accessed_ for resource clicks
* New verb _preferred_ for visua11y clicks
* Adding iterative attempt numbers to question statements

### Breaking
* This upgrade moves to use updated core imports instead of the redundant behavior prior to v5.19.1
* fixes #13 

[//]: # (List appropriate steps for testing if needed)
### Testing
1. MUST have authorization credentials to connect to an established LRS. These credentials can be manually inputted into `required/launch.html`.
2. Enable the plug-in in config.json. See example.json.
3. Navigate through a vanilla course, checking that each statement is appropriately being recorded in the correct store within the LRS.

[//]: # (Mention any other dependencies)
